lib/event_loop.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Loopbench = require('loopbench');
+let sampler = null;
+
+
+module.exports = { start, stop, getState };
+
+
+function start (options) {
+  if (sampler) {
+    return;
+  }
+
+  sampler = Loopbench(options);
```

Comment:
Then no worries, I did not see those lines :(. I'm a bit tired. Let's leave things as they are.